### DISCUSSION TITLE
How to ramp up?

### DISCUSSION THREAD

**Discussion by pushkarajthorat** [helix-editor/helix#459] [Q&A]
I am new to rust and ofcourse helix, want to ramp up on it. Could anyone tell me where should I start - I am currently reading rust.

While going thru the code I saw the code base is of 27k

**Comment by kajuburfi** [comment]
@pickfire told me that you can fork the project, and do whatever changes you are interested in. 
You can try joining the Matrix space of Helix. There you can _also_ discuss with others, as more people see there, than here..

**Reply by pickfire** [reply]
Do what interests you more is the best thing I think. Try playing around with helix and see what you need. If you miss anything from others, try implementing it. If you faced issue, can ping us in matrix.
